### DISCUSSION TITLE
出现了奇怪的问题  Failure to initialize

### DISCUSSION THREAD

**Discussion by IyoJimbo** [chinosk6/gakuen-imas-localify#24] [Q&A] [answered]
![{964D7A61-E431-49b4-B5FD-24218EC77D33}](https://github.com/chinosk6/gakuen-imas-localify/assets/55569896/69c478dc-ab0b-4f21-90d8-5a2182ca8124)

配置是   mixfold2 + hyperos + lspatch + 本地模式  ///尝试过嵌入也是一样的问题

**Comment by chinosk6** [comment]
尝试过直接使用安装包修补吗？

**Reply by IyoJimbo** [reply]
还没有  我试下

**Reply by IyoJimbo** [reply]
依然是同样的报错

**Reply by chinosk6** [reply, marked as answer]
这个报错是你的安装包本身不完整，缺少一些库。你的原安装包是正常的400多MB的吗？

**Reply by IyoJimbo** [reply]
那好像确实不是...我去看看能不能找到完整的安装包

**Reply by chinosk6** [reply]
可以在 [这里](https://apkcombo.com/zh/downloader/#package=com.bandainamcoent.idolmaster_gakuen) 找

**Reply by IyoJimbo** [reply]
确实ok了 感谢

**Reply by IyoJimbo** [reply]
不过之前使用直接修补已用应用时有这个报错  不知道是什么原因

**Reply by chinosk6** [reply]
我在测试的时候使用非目前最新的 LSPatch 版本，会出现这个问题。
在安卓 10 上测试的时候也发现了一些兼容性问题

**Reply by IyoJimbo** [reply]
原来如此  多谢您的耐心帮助
